packages/mui-system/src/breakpoints/breakpoints.js

```diff
@@ -121,10 +121,19 @@ export function createEmptyBreakpointObject(breakpointsInput = {}) {
 
 export function removeUnusedBreakpoints(breakpointKeys, style) {
   return breakpointKeys.reduce((acc, key) => {
+    const zeroMediaQueryRgx = /^@media\s\(min-width:\s?[0](px|rem|em)\)$/;
     const breakpointOutput = acc[key];
     const isBreakpointUnused = !breakpointOutput || Object.keys(breakpointOutput).length === 0;
     if (isBreakpointUnused) {
       delete acc[key];
+    } else if (zeroMediaQueryRgx.test(key)) {
+      const zeroMediaValues = acc[key];
+      delete acc[key];
+      Object.entries(zeroMediaValues).forEach(([styleKey, styleValue]) => {
```

Comment:
This breaks the current UI because it inserts the keys in the zero media query at the end. Instead, it should insert the keys at the same place as the zero media query.
I'll need to find a performant way to do that without using spread operator.

Reply:
I've tried spread operator for now to see if the site still breaks or not.